Dockerfile

```diff
@@ -1,7 +1,20 @@
-FROM scratch
-COPY ./cacert.pem /etc/ssl/certs/ca-certificates.crt
-COPY cardigann-linux-amd64 /cardigann
+FROM alpine:latest
+
+ENV CARDIGANN_PKG cardigann-stable-linux-amd64.tgz
+ENV CONFIG_DIR /etc/cardigann
+
+ADD https://bin.equinox.io/c/3u8U4iwUn6o/${CARDIGANN_PKG} /
+
+# install ca certs
+RUN apk update && apk add ca-certificates
+# make sure the executable can see the shared library
+RUN mkdir /lib64 && ln -s /lib/ld-musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
+# unpack
+RUN tar xfz /${CARDIGANN_PKG} -C /bin/
+# cleanup
+RUN rm -fr /${CARDIGANN_PKG} /root/.cache /tmp/*
```

Comment:
Same deal with the ENV comments, this should be one RUN command, otherwise it results in multiple layers. Try running `docker history <image>` on the result to see what I mean.